plugins/flytekit-inference/README.md

```diff
@@ -67,3 +67,54 @@ def model_serving() -> str:
 
     return completion.choices[0].message.content
 ```
+
+## Ollama
+
+The Ollama plugin allows you to serve LLMs locally.
+You can either pull an existing model or create a new one.
+
+```python
+from flytekit import ImageSpec, Resources, task, workflow
+from flytekitplugins.inference import Ollama, Model
+from flytekit.extras.accelerators import A10G
+from openai import OpenAI
+
+
+image = ImageSpec(
+    name="ollama_serve",
+    registry="...",
+    packages=["flytekitplugins-inference"],
+)
+
+ollama_instance = Ollama(
+    model=Model(
+        name="llama3-mario",
+        modelfile="FROM llama3\nADAPTER {inputs.gguf}\nPARAMETER temperature 1\nPARAMETER num_ctx 4096\nSYSTEM You are Mario from super mario bros, acting as an assistant.",
```

Comment:
For long strings with newlines, I normally like doing this: 

```suggestion
        modelfile=dedent("""\
        FROM llama3
        ADAPTER {inputs.gguf}
        PARAMETER temperature 1
        PARAMETER num_ctx 4096
        SYSTEM You are Mario from super mario bros, acting as an assistant.\
        """),
```

where `from textwrap import dedent`.